app/models/vet360_redis/contact_information.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+require 'vet360/contact_information/person_response'
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+
+module Vet360Redis
+  # Facade for Vet360::ContactInformation::Service. The user_serializer delegates
+  # to this class through the User model.
+  #
+  # When a person is requested from the serializer, it returns either a cached
+  # response in Redis or from the Vet360::ContactInformation::Service.
+  #
+  class ContactInformation < Common::RedisStore
+    include Common::CacheAside
+
+    # Redis settings for ttl and namespacing reside in config/redis.yml
+    #
+    redis_config_key :vet360_contact_info_response
+
+    # @return [User] the user being queried in Vet360
+    #
+    attr_accessor :user
+
+    def self.for_user(user)
+      contact_info      = new
+      contact_info.user = user
+
+      contact_info
+    end
+
+    # Returns the user's email model. In Vet360, a user can only have one
+    # email address.
+    #
+    # @return [Vet360::Models::Email] The user's one email address model
+    #
+    def email
+      return unless @user.loa3?
+
+      value_for('emails')&.first
+    end
+
+    # Returns the user's residence. In Vet360, a user can only have one
+    # residence address.
+    #
+    # @return [Vet360::Models::Address] The user's one residence address model
+    #
+    def address
```

Comment:
I would change this to `residential_address`, and if we need a shortcut for getting the first available address, we can add `address` later.